service/src/allocation_monitor.rs

```diff
@@ -579,7 +369,8 @@ mod tests {
             std::env::var("NETWORK_SUBGRAPH_ID").expect("NETWORK_SUBGRAPH_ID not set");
         let indexer_address = std::env::var("INDEXER_ADDRESS").expect("INDEXER_ADDRESS not set");
         // If you don't specify the correct mnemonic, the attestation signers won't be found
-        let indexer_mnemonic = std::env::var("INDEXER_MNEMONIC").expect("INDEXER_MNEMONIC not set");
+        let _indexer_mnemonic =
```

Comment:
Can this be removed?

Reply:
Yes! Actually I don't remember adding the underscore there, weird.

Reply:
Fix in a410943128cabe8166880b041bb287714e770db0